metricbeat/module/uwsgi/_meta/config.yml

```diff
@@ -0,0 +1,7 @@
+- module: uwsgi
+  metricsets: ["status"]
+  enabled: false
```

Comment:
Could you remove `enabled` here? we use the default now (true), as this is handled by `modules` subcommand